src/app/modules/consultations/consultation-profile/read-respond/read-respond.component.html

```diff
@@ -206,130 +206,202 @@
                     </div>
                   </div>
                   <span  
-                  class="ml-4 mr-2 fs-20" >
-                  <i class="fa fa-share pointer" (click)="toggleShareBlock(response?.node?.id)" tooltip="{{'Share'}}" #shareButtonElement></i>
+                  class="fs-20">
+                  <i class="fa fa-share"></i>
+                </span>
+                <span *ngIf="response.node?.templatesCount">
+                  {{response.node?.templatesCount ? response.node?.templatesCount : '' }}
                 </span>
-                  <span>{{response.node?.templatesCount ? response.node?.templatesCount : '' }}</span>
               </div>
             </div>
             <div *ngIf="showCreateResponse()">
-              <button class="app-btn response-btn" (click)="useThisResponse(response?.node)" >{{'Use this response' | translate}}</button>
+              <button class="app-btn response-btn"
+                (click)="useThisResponse(response?.node)">{{'Use this response' | translate}}</button>
             </div>
           </div>
         </div>
       </ng-container>
     </div>   
-  </ng-container>   
+  </div>   
 </section>
 <div *ngIf="selectedUser && showLeaderProfileModal">
   <app-user-profile-modal (close)="closeModal($event)" [userId]="selectedUser"></app-user-profile-modal>
 </div>
 
 <!-- rating-model -->
 <div class="feedback-index">
```

Comment:
@Nishanthh What was changed here? Only UI?

Reply:
@Nishanthh Can you answer this question?

Reply:
@avikt Yes only  UI.